### DISCUSSION TITLE
手動添加的詞庫和使用 rime_dict_manager 添加的詞庫有什麼不同？

### DISCUSSION THREAD

**Discussion by resigned1292** [rime/home#986] [Q&A] [answered]
Rime 用戶可以通過新建 *.dict.yaml 文件來添加詞庫，也可以通過 rime_dict_manager 命令行工具來添加詞庫，那麼這兩種方式添加的詞庫有什麼區別呢？爲什麼建議手動添加「外來碼錶」？

**Comment by lotem** [comment, marked as answer]
前者編譯爲固態詞典。固態詞典提供詞彙表及字詞的編碼。
後者合併到用戶詞典。用戶詞典通過記錄上屏的文字學習用戶的用詞習慣。
外來碼表，顯然不是用戶的輸入記錄，而是帶編碼的詞彙表。

**Reply by resigned1292** [reply]
那么更推荐使用哪种方式添加词库呢？
